projects/cdk/schematics/ng-update/steps/migrate-input-files.ts

```diff
@@ -0,0 +1,99 @@
+import {
```

Comment:
try add example to `https://github.com/Tinkoff/taiga-ui/blob/main/projects/cdk/schematics/ng-update/tests/schematic-replace-html.spec.ts`

Reply:
Committed!

Reply:
I can see, that there is a conflict with the `schematic-replace-html.spec.ts` file. Can I resolve it?

Reply:
rebase with main branch